crates/fluvio-cli/src/consume/mod.rs

```diff
@@ -584,11 +588,48 @@ impl ConsumeOpt {
             }
             (_, Some(templates)) => {
                 let value = String::from_utf8_lossy(record.value()).to_string();
+                let timestamp_rfc3339 = if record.timestamp() == NO_TIMESTAMP {
+                    "NA".to_string()
+                } else {
+                    format!(
+                        "{}",
```

Comment:
Instead of hard coding time values, can we use formatted time like here? https://rust-lang-nursery.github.io/rust-cookbook/datetime/parse.html.  We can also treat human time similarly

Reply:
We can, I just didn't want to add chronos as dependency, but if that is not a problem, I can proceed with that library

Reply:
Is that better than the actual RFC format?

I also can extract year, month, day, hour, minute and second, and then we could do something like:

`"{{year}}-{{month}}-{{day}}-{{hour}}-{{min}}-{{sec}}"`

Reply:
Can we leverage https://docs.rs/chrono/0.4.19/chrono/format/strftime/index.html#specifiers?

Reply:
Also, it would probably be better to split this PR since formatting make needs additional refinement.  We can merge API-related changes first.